composer.json

```diff
@@ -16,13 +16,13 @@
     ],
     "require": {
         "php": ">=7.2",
-        "psr/http-message": "^1.0",
+        "psr/http-message": "^1.1 || ^2.0",
         "php-http/message-factory": "^1.0",
         "psr/http-factory": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^7.5 || 8.5 || 9.4",
-        "php-http/psr7-integration-tests": "^1.0",
+        "php-http/psr7-integration-tests": "^1.0@dev",
```

Comment:
This line looks desirable to me: testing with `@dev` for the test suite means anticipating any unreleased issue with them.